challenge_storing_groceries/src/challenge_storing_groceries/storing_groceries.py

```diff
@@ -43,34 +44,38 @@ def __init__(self, robot):
                                                 'no_response': 'AWAIT_START'})
 
             cabinet = ds.EntityByIdDesignator(robot, id=CABINET)
-            room = ds.EntityByIdDesignator(robot, id=ROOM)
+            # room = ds.EntityByIdDesignator(robot, id=ROOM)
```

Comment:
If this room is not required, it doesn't need to be commented out, right?

Reply:
Done in 0fb7608e438c92a8c3522274d2880d9b837ef0a5